src/main/java/org/marketdesignresearch/mechlib/mechanism/auctions/interactions/ValueQuery.java

```diff
@@ -8,4 +8,7 @@
 
 public interface ValueQuery<B extends BundleValueBid<?>> extends TypedInteraction<B> {
 	Set<Bundle> getQueriedBundles();
+	default Bundle getAlreadyWon() {
+		return Bundle.EMPTY;
+	}

```

Comment:
I'm not sure if we should add such specific properties to a ValueQuery. In its raw form a Value Query as used in MLCA is just:
Report your values for the given set of bundles.

Therefore I would rather suggest to implement a different Interaction for the sequential auction.